### PR TITLE
Translation in pa-IN.

### PR DESCRIPTION
**Pull request by Sahilbal**

### CODE REVIEW

File: assets/translations/pa-IN.json [hunk 1]
```diff
@@ -2,22 +2,22 @@
...
   "Empty_history": "ਖਾਲੀ ਇਤਿਹਾਸ",
-  "History_Settings_Page": "ਇਤਿਹਾਸ ਸੈਟਿੰਗਾਂ ਪੰਨਾ",
-  "Lamps_Page": "ਲੈਂਪਸ ਪੰਨਾ",
-  "Name:_": "ਨਾਮ: {}",
-  "Room:_": "ਕਮਰਾ: {}",
-  "Lamps_Settings_Page": "ਲੈਂਪਸ ਸੈਟਿੰਗਾਂ ਪੰਨਾ",
+  "History_Settings_Page": "ਇਤਿਹਾਸ ਸੈਟਿੰਗ ਪੰਨਾ",
+  "Lamps_Page": "ਦੀਵਾ  ਪੰਨਾ ",
+  "Name:_": "ਨਾਮ  : {}",
+  "Room:_": " ਕਮਰਾ: {}",
```
Comment (guyluz11) [inline assets/translations/pa-IN.json:8]: Extra space please delete it

File: assets/translations/pa-IN.json [hunk 2]
```diff
@@ -2,22 +2,22 @@
...
   "Actions_History": "ਕਾਰਵਾਈਆਂ ਦਾ ਇਤਿਹਾਸ",
   "Empty_history": "ਖਾਲੀ ਇਤਿਹਾਸ",
-  "History_Settings_Page": "ਇਤਿਹਾਸ ਸੈਟਿੰਗਾਂ ਪੰਨਾ",
-  "Lamps_Page": "ਲੈਂਪਸ ਪੰਨਾ",
-  "Name:_": "ਨਾਮ: {}",
-  "Room:_": "ਕਮਰਾ: {}",
-  "Lamps_Settings_Page": "ਲੈਂਪਸ ਸੈਟਿੰਗਾਂ ਪੰਨਾ",
+  "History_Settings_Page": "ਇਤਿਹਾਸ ਸੈਟਿੰਗ ਪੰਨਾ",
+  "Lamps_Page": "ਦੀਵਾ  ਪੰਨਾ ",
+  "Name:_": "ਨਾਮ  : {}",
```
Comment (guyluz11) [inline assets/translations/pa-IN.json:7]: Extra space before the `:` please delete it

File: assets/translations/pa-IN.json [hunk 3]
```diff
@@ -2,22 +2,22 @@
   "Create_new_account": "ਨਵਾਂ ਖਾਤਾ ਬਣਾਓ",
   "Actions_History": "ਕਾਰਵਾਈਆਂ ਦਾ ਇਤਿਹਾਸ",
   "Empty_history": "ਖਾਲੀ ਇਤਿਹਾਸ",
-  "History_Settings_Page": "ਇਤਿਹਾਸ ਸੈਟਿੰਗਾਂ ਪੰਨਾ",
-  "Lamps_Page": "ਲੈਂਪਸ ਪੰਨਾ",
-  "Name:_": "ਨਾਮ: {}",
-  "Room:_": "ਕਮਰਾ: {}",
-  "Lamps_Settings_Page": "ਲੈਂਪਸ ਸੈਟਿੰਗਾਂ ਪੰਨਾ",
+  "History_Settings_Page": "ਇਤਿਹਾਸ ਸੈਟਿੰਗ ਪੰਨਾ",
+  "Lamps_Page": "ਦੀਵਾ  ਪੰਨਾ ",
```
Comment (guyluz11) [inline assets/translations/pa-IN.json:6]: 2 spaces between the worlds, please keep one.
And one extra space in the end.

File: assets/translations/pa-IN.json [hunk 4]
```diff
@@ -2,22 +2,22 @@
...
+  "Name:_": "ਨਾਮ  : {}",
+  "Room:_": " ਕਮਰਾ: {}",
+  "Lamps_Settings_Page": "ਦੀਵਾ ਸੈਟਿੰਗਾਂ ਪੰਨਾ",
   "Welcome_to_your_Smart_Home": "ਤੁਹਾਡੇ ਸਮਾਰਟ ਹੋਮ ਵਿੱਚ ਸੁਆਗਤ ਹੈ",
   "Scene_Settings_Page": "ਦ੍ਰਿਸ਼ ਸੈਟਿੰਗਾਂ ਪੰਨਾ",
-  "Move_to_blinds_full_settings_page": "ਬਲਾਇੰਡਸ ਪੂਰੇ ਸੈਟਿੰਗਾਂ ਪੰਨੇ 'ਤੇ ਜਾਓ",
-  "Blinds_Page": "ਬਲਾਇੰਡਸ ਪੰਨਾ",
-  "Down": "ਥੱਲੇ, ਹੇਠਾਂ, ਨੀਂਵਾ",
+  "Move_to_blinds_full_settings_page": "ਪਰਦੇ ਦੇ ਪੂਰੇ ਸੈਟਿੰਗ ਪੰਨੇ 'ਤੇ ਜਾਓ",
+  "Blinds_Page": " ਪਰਦਾ ਪੰਨਾ",
```
Comment (guyluz11) [inline assets/translations/pa-IN.json:13]: Extra space please delete it

File: assets/translations/pa-IN.json [hunk 5]
```diff
@@ -30,44 +30,83 @@
...
   "Cancel": "ਰੱਦ ਕਰੋ",
   "Add": "ਸ਼ਾਮਲ ਕਰੋ",
   "Light": "ਚਾਨਣ",
-  "Device_ip:_": "ਡਿਵਾਈਸ ip: {}",
+  "Device_ip:_": "ਡਿਵਾਈਸ ਆਈ.ਪੀ: {}",
   "New_devices": "ਨਵੀਆਂ ਡਿਵਾਈਸਾਂ",
-  "Add_devices": "ਡਿਵਾਈਸਾਂ",
-  "Device_type:_": "ਨਵੀਆਂ: {}",
+  "Add_devices": "ਡਿਵਾਈਸਾਂ ਸ਼ਾਮਲ ਕਰੋ",
+  "Device_type:_": "ਡਿਵਾਈਸ ਦੀ ਕਿਸਮ : {}",
```
Comment (guyluz11) [inline assets/translations/pa-IN.json:53]: Extra space before `:` please delete it

File: assets/translations/pa-IN.json [hunk 6]
```diff
@@ -30,44 +30,83 @@
...
   "Room_name_is_required": "ਕਮਰੇ ਦਾ ਨਾਮ ਲੋੜੀਂਦਾ ਹੈ",
   "Room_name:": "ਕਮਰੇ ਦਾ ਨਾਮ:",
   "Device_name_is_required": "ਡਿਵਾਈਸ ਦਾ ਨਾਮ ਲੋੜੀਂਦਾ ਹੈ",
-  "Device_name:": "ਡਿਵਾਈਸ ਦਾ ਨਾਮ:",
+  "Device_name:": "ਡਿਵਾਈਸ ਦਾ ਨਾਮ :",
```
Comment (guyluz11) [inline assets/translations/pa-IN.json:57]: Extra space before ":" please delete it

File: assets/translations/pa-IN.json [hunk 7]
```diff
@@ -30,44 +30,83 @@
...
   "Update_WiFi_configuration": "ਵਾਈ-ਫਾਈ ਕੌਂਫਿਗਰੇਸ਼ਨ ਅੱਪਡੇਟ ਕਰੋ",
-  "WiFi_name_is_required": "WiFi ਨਾਮ ਦੀ ਲੋੜ ਹੈ",
-  "WiFi_Name": "WiFi ਨਾਮ",
-  "WiFi_password_is_required": "WiFi ਪਾਸਵਰਡ ਦੀ ਲੋੜ ਹੈ",
-  "WiFi_Password": "WiFi ਪਾਸਵਰਡ",
-  "Nothing": "ਕੁਝ ਨਹੀਂ",
-  "Finish": "ਸਮਾਪਤ",
-  "On": "'ਤੇ",
+  "WiFi_name_is_required": "ਵਾਈਫਾਈ ਨਾਮ ਦੀ ਲੋੜ ਹੈ",
+  "WiFi_Name": "ਵਾਈਫਾਈ  ਨਾਮ",
```
Comment (guyluz11) [inline assets/translations/pa-IN.json:64]: You have double space

File: assets/translations/pa-IN.json [hunk 8]
```diff
@@ -30,44 +30,83 @@
...
+  "WiFi_password_is_required": "ਵਾਈਫਾਈ  ਪਾਸਵਰਡ ਦੀ ਲੋੜ ਹੈ",
```
Comment (guyluz11) [inline assets/translations/pa-IN.json:65]: You have double space

File: assets/translations/pa-IN.json [hunk 9]
```diff
@@ -30,44 +30,83 @@
...
+  "WiFi_Password": "ਵਾਈਫਾਈ  ਪਾਸਵਰਡ",
```
Comment (guyluz11) [inline assets/translations/pa-IN.json:66]: You have double space

File: assets/translations/pa-IN.json [hunk 10]
```diff
@@ -30,44 +30,83 @@
...
+  "Summary": "ਸੰਖੇਪ",
+  "Switches": "ਸਵਿੱਚ",
+  "Boilers": "ਬਾਇਲਰ",
+  "Lights": "ਲਾਈਟਾਂ",
+  "RGBw_Lights": "RGBw ਲਾਈਟਾਂ",
+  "_Switches": "{} ਸਵਿੱਚ",
+  "_SmartTvs": "{} ਸਮਾਰਟ ਟੀ.ਵੀ",
+  "_RGBW_Lights": "{} RGBW ਲਾਈਟਾਂ",
+  "_Lights": "{} ਲਾਈਟਾਂ",
+  "_Boilers": "{}ਬਾਇਲਰ ",
```
Comment (guyluz11) [inline assets/translations/pa-IN.json:83]: Space is needed after `}`

File: assets/translations/pa-IN.json [hunk 11]
```diff
@@ -30,44 +30,83 @@
...
+  "No_need_here": "ਇੱਥੇ ਕੋਈ ਲੋੜ ਨਹੀਂ",
+  "Do_not_need": "ਲੋੜ ਨਹੀਂ ਹੈ",
+  "Finish": "ਸਮਾਪਤ",
+  "Select_Purposes_Of_The_Area": "ਖੇਤਰ ਦੇ ਉਦੇਸ਼ਾਂ ਦੀ ਚੋਣ ਕਰੋ",
+  "Pulling_Up_all_blinds": "ਸਾਰੇ ਪਰਦੇ ਨੂੰ ਖਿੱਚਣਾ",
+  "Stopping_all_blinds": "ਸਾਰੇ ਬਲਾਇੰਡਸ ਨੂੰ ਰੋਕ ਰਿਹਾ ਹੈ",
+  "Pulling_down_all_blinds": "ਸਾਰੇ ਪਰਦੇ ਹੇਠਾਂ ਖਿੱਚ ਰਹੇ ਹਨ",
+  "Turning_On_boiler": "ਬਾਇਲਰ ਨੂੰ ਚਾਲੂ ਕਰਨਾ",
+  "Turning_Off_boiler": "ਬਾਇਲਰ ਨੂੰ ਬੰਦ ਕਰਨਾ",
+  "Waiting_for_user_to_get_connected_to_WiFi": "ਉਪਭੋਗਤਾ ਦੇ  ਵਾਈਫਾਈ  ਨਾਲ ਕਨੈਕਟ ਹੋਣ ਦੀ ਉਡੀਕ ਕੀਤੀ ਜਾ ਰਹੀ ਹੈ",
```
Comment (guyluz11) [inline assets/translations/pa-IN.json:110]: 2 double spaces please fix.

**Review by guyluz11 — CHANGES_REQUESTED** [review]

File: assets/translations/pa-IN.json [hunk 12]
```diff
@@ -80,7 +80,7 @@
   "_SmartTvs": "{} ਸਮਾਰਟ ਟੀ.ਵੀ",
   "_RGBW_Lights": "{} RGBW ਲਾਈਟਾਂ",
   "_Lights": "{} ਲਾਈਟਾਂ",
-  "_Boilers": "{}ਬਾਇਲਰ ",
+  "_Boilers": "{} ਬਾਇਲਰ ",
```
Comment (guyluz11) [inline assets/translations/pa-IN.json:83]: Extra space in the end

**Review by guyluz11 — CHANGES_REQUESTED** [review]
I think this is the only one needed now

File: assets/translations/pa-IN.json (hunk 5 above)
Comment (Sahilbal) [reply assets/translations/pa-IN.json]: okay got it!. thank you.

**Comment by Sahilbal** [comment]
I think I have made all the required changes . So please accept my pull request and give it hactoberfest accepted tag. kindly let me know if I am still lagging to meet your requirements.  I shall be very thankful to you. Also you are a best teacher I ever met because our teachers don't guide us  like you. They get annoyed very often when we ask them the same thing second time.

File: assets/translations/pa-IN.json [hunk 13]
```diff
@@ -2,22 +2,22 @@
   "Create_new_account": "ਨਵਾਂ ਖਾਤਾ ਬਣਾਓ",
   "Actions_History": "ਕਾਰਵਾਈਆਂ ਦਾ ਇਤਿਹਾਸ",
   "Empty_history": "ਖਾਲੀ ਇਤਿਹਾਸ",
-  "History_Settings_Page": "ਇਤਿਹਾਸ ਸੈਟਿੰਗਾਂ ਪੰਨਾ",
-  "Lamps_Page": "ਲੈਂਪਸ ਪੰਨਾ",
+  "History_Settings_Page": "ਇਤਿਹਾਸ ਸੈਟਿੰਗ ਪੰਨਾ",
+  "Lamps_Page": "ਦੀਵਾ ਪੰਨਾ ",
```
Comment (guyluz11) [inline assets/translations/pa-IN.json:6]: Extra space in the end

**Review by guyluz11 — CHANGES_REQUESTED** [review]

**Comment by Sahilbal** [comment]
I have made all the required changes. So please accept my pull request and give it hactoberfest accepted tag.

**Review by guyluz11 — APPROVED** [review]
Nice 👍

**Comment by guyluz11** [comment]
Thanks for the pr @Sahilbal and happy Hacktoberfest 😁

**Comment by Sahilbal** [comment]
> Thanks for the pr @Sahilbal and happy Hacktoberfest 😁

thank you!

**Pull request merged**